bedrock/mozorg/templates/mozorg/technology.html

```diff
@@ -29,6 +29,7 @@
 {% endblock %}
 
 {% block content %}
+{% include 'mozorg/developer/includes/sub-nav.html' %}
```

Comment:
I believe this page is localized, so it may need some for of l10n tag (cc @flodolo)

Reply:
The sub nav on this page also isn't sticky, so it's missing some JS.

Reply:
Ah yes, I did not specifically notice that but, was wondering whether there was some JS required. Will add in what is missing.

Reply:
Ok so, maybe I should create a new one called `dev-tech-sub-nav.html`. Then I can do:

```
{% if LANG.startswith('en-') %}
 new subnav
{% else %}
  current nav
{% endif %}
```

Reply:
Not quite sure i follow, why would be need a new sub nav? I think an l10n tag should be enough, but I'll defer to @flodolo

Reply:
ah ok, I might misunderstand what an l10n flag is :-/ I was thinking we might want to show the new navigation items only to `en-*` until it has been localized in enough locales. While it is being localized, we show the current entries to everyone else.